octomachinery/github/entities/app_installation.py

```diff
@@ -53,11 +53,13 @@ def token(self):
     async def retrieve_access_token(self):
         """Retrieve installation access token from GitHub API."""
         async with self._github_app.github_app_client as gh_api:
-            self._token = GitHubInstallationAccessToken(**(await gh_api.post(
+            result = await gh_api.post(
                 self._metadata.access_tokens_url,
                 data=b'',
                 accept='application/vnd.github.machine-man-preview+json',
-            )))
+            )
+            self._token = GitHubInstallationAccessToken(token=result['token'],
```

Comment:
Looks like a band-aid... Maybe it needs to be redesigned but for now, I'd definitely want to see the new structure reflected in the model class.